src/app-modules/docker-compose

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+set -x
+
+docker_compose_cmd=docker-compose
+docker_cmd=docker
+
+app_load() {
+    local url="$1"
+    local input_file="$2"
+
+    $docker_cmd image load < "$input_file"
+}
+
+app_save() {
+    local url="$1"
+    local output_file="$2"
+
+    $docker_cmd image save "$url" -o "$output_file"
+}
+
+app_rollout() {
+    local manifests_dir="$1"
+
+    (cd "$manifests_dir" && $docker_compose_cmd up -d > compose.log 2>&1 3>&1) &
+}
+
+app_stop() {
+    local manifests_dir="$1"
+
+    (cd "$manifests_dir" && $docker_compose_cmd down)
+}
+
+case "$1" in
+    save)
```

Comment:
The [API docs](https://github.com/mendersoftware/app-update-module/blob/master/docs/README-submodule-api.md) seems to indicate that these should be uppercase.

Reply:
fixed.

Reply:
But do we really want it in uppercase? I thought you would fix the docs instead :sweat_smile: 

It is good with me both ways, as long as doc and code say the same thing